src/cljs/ataru/virkailija/editor/subs.cljs

```diff
@@ -163,6 +165,29 @@
                    (clojure.string/split name
                                          (re-pattern (str "(?i)(" search-term ")"))))
       [[name false]])))
+(re-frame/reg-sub
+  :editor/unique-ids-in-form
+  (fn [db _]
+    (not-empty
```

Comment:
Ottasin ehkä tästä `not-empty` pois ja kysyisin tuolla `editor/can-copy-or-paste?`:ssa `empty?`.